src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.spec.ts

```diff
@@ -82,9 +84,15 @@ describe('OsdListComponent', () => {
       CephModule,
       ReactiveFormsModule,
       RouterTestingModule,
-      CoreModule
+      CoreModule,
```

Comment:
Having the imports and declarations like the following will fix the unit tests errors:
```
    imports: [
      HttpClientTestingModule,
      PerformanceCounterModule,
      TabsModule.forRoot(),
      ReactiveFormsModule,
      RouterTestingModule,
      SharedModule
    ],
    declarations: [
      OsdListComponent,
      OsdDetailsComponent,
      OsdPerformanceHistogramComponent,
      SmartListComponent,
      DeviceListComponent
    ],
```